guides/v2.2/extension-dev-guide/package/package_module.md

```diff
@@ -51,7 +51,7 @@ The following example is a `composer.json` for a metapackage:
     "version": "2.0.0",
     "type": "metapackage",
     "require": {
-        "php": "~5.5.0|~5.6.0|~7.0.0",
+        "php": "~7.1.3|~7.2.0",
```

Comment:
It is different for _magento2_ v2.2 though:
https://github.com/magento/magento2/blob/2.2/composer.json#L11

Reply:
thanks for catching this @dshevtsov. I'll break the symlink and restore the 2.2 version.

Reply:
I almost said something about this, but I decided against, since it's unlikely anyone is developing new modules for 2.2 at this point.